src/Controller/UI/SearchPageController.php

```diff
@@ -23,14 +23,31 @@ class SearchPageController extends AbstractController
      */
     protected $logActionItemEventDispatcher;
 
+    /**
+     * Subsite name, based on custom template.
+     *
+     * @var string
+     */
+    protected $subSite;
+
     /**
      * Constructor for this Controller, to set up default services.
      *
      * @param LogActionItemEventDispatcher $logActionItemEventDispatcher The log action item event dispatcher.
      */
-    public function __construct(LogActionItemEventDispatcher $logActionItemEventDispatcher)
+    public function __construct(LogActionItemEventDispatcher $logActionItemEventDispatcher, ?string $customTemplate)
     {
         $this->logActionItemEventDispatcher = $logActionItemEventDispatcher;
+        if (empty($customTemplate)) {
+            // If custom template is not set, subSite is 'GRIIDC'.
```

Comment:
Maybe this should be always set to something, but that would be an improvement we might consider some day.